step-plans/step-plans-core/pom.xml

```diff
@@ -44,6 +44,16 @@
 			<artifactId>step-functions-client</artifactId>
 			<version>${project.version}</version>
 		</dependency>
+		<dependency>
+			<groupId>ch.exense.step</groupId>
+			<artifactId>step-api-json-schema</artifactId>
+			<version>${step-api.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.reflections</groupId>
+			<artifactId>reflections</artifactId>
+			<version>0.10.2</version>
+		</dependency>

```

Comment:
@jeromecomte 
I was used, because intially I wanted to find some classes (Migration tasks for yaml format) not by annotations but by parent class. But I've reworked this and added the explicit annotation (@YamlPlanMigration) to mark these migrations.
Now I use the AnnotationScanner (dependency on org.reflections has been removed).